### ISSUE TITLE
Certain names are removed from the expiration list moments before while most stay on the list after expiration

### ISSUE THREAD

**Issue by GitFist** [bricefrisco/NamesLoL#15]

**Comment by bricefrisco** [comment]
Hey @GitFist, thanks for the report.

The only reason (that I am aware of) that a name will be removed from the list is if someone else took the name. Once another player takes a name, it will be bumped far down the list as the player's activity date has then been reset.

This can be validated by searching for a name using the name checker to see if the expiration date has moved into the future. If you have a specific example, I can look into this as well.

**Comment by GitFist** [comment]
Thank you for the response, I was trying to grab the name 'Druid' which was set to expire 05/22/2022 07:08:37 PM. I had two tabs open, one specifically looking at 'Druid' on the Name Checker page and one looking at where 'Druid' should be on the Expiration List.
I noticed maybe 10 minutes prior to the expiration that the name was removed from the Expiration List.
After 07:08:37 PM, I refreshed the Name Checker page and it was set to expire now on 5/22/2023 at 07:08:37

TL;DR
The name Druid was missing from the expiration list minutes before it actually expired and was taken at the exact time it was set to expire.

Sorry if I overcomplicated it!
